rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
@yifan-gu Do you intend to rename the option to be consistent with `metadata-service`'s `--listen-port=` as well?

It seems reasonable to me to separate port binding from name/IP, and now is the time to break old clients rather than later, but probably we should run it by @jonboulle